src/Symfony/Component/HttpKernel/DataCollector/ConfigDataCollector.php

```diff
@@ -219,7 +219,7 @@ public function getEnv()
     /**
      * Returns true if the debug is enabled.
      *
-     * @return bool true if debug is enabled, false otherwise
+     * @return bool|string true if debug is enabled, false otherwise or a string if no kernel was set
```

Comment:
Is this better?

```suggestion
     * @return bool|'n/a' true if debug is enabled, false otherwise or a string if no kernel was set
```

Reply:
`n/a` is not a valid type I think

Reply:
> `n/a` is not a valid type I think

`'n/a'` is a literal type. PHPStan and Psalm support literal types, but I'm not aware that we've ever used them in the Symfony codebase.